lib/services/iotaRedirector.js

```diff
@@ -209,7 +216,7 @@ function processRequests(req, res, next) {
 
             if (statusCodes.length === 1) {
                 if (results.length === 1) {
-                    res.status(statusCodes[0]).json(results[0]);
+                    res.status(statusCodes[0]).json(results[0][1]);
```

Comment:
How sure are you about results is a matrix? Already results[0] is checked, but results[0][1] exists?